man/cve-publish.1

```diff
@@ -1,25 +1,29 @@
-.TH "CVE PUBLISH" "1" "2022-04-20" "0.7.0" "cve publish Manual"
+.TH "CVE PUBLISH" "1" "2022-09-30" "1.0.0" "cve publish Manual"
 .SH NAME
-cve\-publish \- Publish a CVE record for an...
+cve\-publish \- Publish a CVE record for a reserved (or...
 .SH SYNOPSIS
 .B cve publish
 [OPTIONS] CVE_ID
 .SH DESCRIPTION
-Publish a CVE record for an already-reserved CVE ID.
+Publish a CVE record for a reserved (or rejected) CVE ID.
 .PP
-Will not update if the CVE record is already published.
+    If the CVE is already published, this action will update its record. A published CVE can only be
+    moved to the rejected state with an appropriate reject record (see `cve reject`). A published
+    CVE cannot be moved back to the reserved state.
 .PP
-
-cve publish 'CVE-2022-1234' --json \
-'{"affected": [], "descriptions": [], "providerMetadata": {}, "references": []}'
+    Example:
 .PP
-For information on the required properties in a given CVE JSON record, see the
-`cnaPublishedContainer` schema in:
+    
+    cve publish CVE-2022-1234 -j '{"affected": [], "descriptions": [], "references": {}, ...}'
```

Comment:
Not sure why this is changing - are these docs autogenerated? One of the keys (maybe "orgName"?) in "providerMetadata"' is required, but "providerMetadata" is no longer shown here. I included only the required top-level keys in this example. It also looks like this got truncated, even though this is the full-text part of the man page and not the summary / title / short description.

Reply:
That was already truncated in the previous PR to make it fit within 100 chars without a `\`. I don't think it needs to present a valid record, nobody is expected to copy this out and use it to publish a record. It's just there as an example of what the input might kind of look like.